src/transformers/file_utils.py

```diff
@@ -1903,6 +1903,9 @@ def http_user_agent(user_agent: Union[Dict, str, None] = None) -> str:
 def http_get(url: str, temp_file: BinaryIO, proxies=None, resume_size=0, headers: Optional[Dict[str, str]] = None):
     """
     Download remote file. Do not gobble up errors.
+
+    `tqdm` behavior is determined by `transformers.utils.tqdm_utils.is_progress_bar_enabled`, and can be set using
+    `transformers.utils.tqdm_utils.set_progress_bar_enabled`.
```

Comment:
I suggested that Jake adds it here, because if you look at the original it had a way (albeit broken) of changing the behavior. 

So we aren't changing anything, just replacing the old broken behavior with a new working one and documenting that new way.

Reply:
The documentation is only in the end function, so if you want to document it so a user can find it, it would need to go in every function that ends up calling this one. I'd keep things simple and just remove this.

Reply:
Here is how I think about it: `http_get` is the function that invokes tqdm, and not its callers. Therefore a user tracing the tqdm call will find that `http_get` calls `tqdm`, and read the doc that explains how to control this behavior.

If this note is removed now the user will have no idea how to control this behavior, unless they think to go to the logging doc and read it there, which they may or may not think of it, since tqdm doesn't use logging.

Reply:
What you're describing would be better achieved with a comment in the code, not the dosctring.

Reply:
That works. Let's do that.